content/departments/product-engineering/process/cross-org-team-collab.md

```diff
@@ -0,0 +1,19 @@
+# Cross-org/team collaboration
+
+Sometimes, ideas (and even code) for something your team owns comes from other teams, orgs, or even departments at Sourcegraph. That's a good thing because everyone can contribute to the innovation and success of the product we all use, and the more we do of that sort of thing, the more of a flywheel we're generating for improvements and is one of the real advantages of being a software development company that makes products for software developers.
```

Comment:
```suggestion
Sometimes, ideas (and even code) for something your team owns comes from other teams, orgs, or even departments at Sourcegraph. That's a good thing because everyone can contribute to the innovation and success of the product we all use, and the more we do of that sort of thing, the more of a flywheel we're generating for improvements and that is one of the real advantages of being a software development company that makes products for software developers.
```